gsensord_test.go

```diff
@@ -0,0 +1,41 @@
+package nmea
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var gsensoredtests = []struct {
+	name string
+	raw  string
+	err  string
+	msg  GSensord
+}{
+	{
+		name: "good sentence",
+		raw:  "$GSENSORD,0.060,0.060,-0.180",
+		msg: GSensord{
+			X: 0.06,
+			Y: 0.06,
+			Z: -0.180,
+		},
+	},
+}
+
+func TestGSensord(t *testing.T) {
```

Comment:
can you add tests cases for other, non-valid, sentences? E.g. sending non-float values to x,y,z; more arguments than expected, including checksum...